cleanrl/ppo_dna_atari_envpool.py

```diff
@@ -0,0 +1,468 @@
+# DNA: Proximal Policy Optimization with a Dual Network Architecture
+# https://arxiv.org/abs/2206.10027
+
+import argparse
+import os
+import random
+import time
+from collections import deque
+from distutils.util import strtobool
+
+import envpool
+import gym
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.optim as optim
+from torch.distributions import Categorical, kl_divergence
+from torch.utils.tensorboard import SummaryWriter
+
+
+def parse_args():
+    # fmt: off
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--exp-name", type=str, default=os.path.basename(__file__).rstrip(".py"),
+        help="the name of this experiment")
+    parser.add_argument("--seed", type=int, default=np.random.randint(2 ** 31),
```

Comment:
Please revert the default seed to 1. This is an expected behavior of using CleanRL that the seeds by default is the same for reproducibility and debugging reason.

Reply:
👍